prow/deck/jobs/jobs.go

```diff
@@ -109,6 +110,19 @@ type filteringProwJobLister struct {
 	hiddenRepos func() sets.String
 	hiddenOnly  bool
 	showHidden  bool
+	tenantIDs   []string
+}
+
+func (c *filteringProwJobLister) TenantIDMatch(pj prowapi.ProwJob) bool {
+	if pj.Spec.ProwJobDefault == nil {
```

Comment:
No. Default tenantIDs should be explicitly defined. Otherwise as soon as this PR landed all of the Prowjobs would be given a defaultTenantID and not appear on any Deck instance.